tests/templates/kuttl/mount-dags-gitsync/dag_metrics.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python
+
+import requests
+import time
+import sys
+
+
+def exception_handler(exception_type, exception, traceback):
```

Comment:
I ran the tests locally and this script failed but I saw no error message. My suggestion would be to:
* get rid of this handler
* `import logging` and add some `logging.error()` and `logging.info()` calls
* use `try/catch/finally` in the loop below

Reply:
As discussed.